spec/models/manageiq/providers/openshift/container_manager/refresher_spec.rb

```diff
@@ -134,6 +134,14 @@ def normal_refresh
     # and the following commands for 3 projects my-project-X (X=0/1/2):
     # oc new-project my-project-X
     # oc process -f template.yml -v INDEX=X | oc create -f -
+    # oc start-build my-build-config-X
+    #
+    # Or this:
```

Comment:
:+1: for the bash version
(Surly we speak it better the English :smirk: )

Other then that PR looks good